martypy/Marty.py

```diff
@@ -251,6 +251,20 @@ def get_ready(self, blocking: Optional[bool] = None) -> bool:
             self.client.wait_if_required(4000, blocking)
         return result
 
+    def stand_straight(self, blocking: Optional[bool] = None) -> bool:
```

Comment:
@alexanderEnoch shouldn't we include `move_time` here as well? It is configurable in Scratch, isn't it?
```suggestion
    def stand_straight(self, move_time: int = 2000, blocking: Optional[bool] = None) -> bool:
```

Reply:
Yes, we should!

Reply:
yes scratch has it configurable :) so I agree with this